dev-tools/ecs-migration.yml

```diff
@@ -587,6 +587,11 @@
   to: http.version
   alias: true
 
+# Filebeat Redis Input
+
+- from: read_timestamp
+  to: event.created
+  alias: false
```

Comment:
Why alias false? I think we used to use `read_timestamp` in a few places. That could be a 1:1 mapping that catches most cases.

Reply:
For now it's false as there are still a few modules which use `read_timestamp` as a field. As soon as we completed the migration, we can probably make it an alias. Will add a checkbox to our migration issue.